modules/core/shared/src/main/scala/lucuma/core/math/validation/MathValidators.scala

```diff
@@ -53,7 +53,7 @@ trait MathValidators {
       a =>
         f"${truncateAngle(a).toSignedBigDecimalDegrees}%.2f"
           .replace("-0.00", "0.00")
-          .replace("180.00", "-180.00")
+          .replace("$180.00", "-180.00")
```

Comment:
What is the `$` for? Is there actually a `$` in the original string?

Reply:
Ah. I guess the old replace was changing "-180.00" to "--180.00"? But, the new replace doesn't seem to fix that, to me.

Reply:
exactly the `--180` triggered a test error. `replace` takes a regex

Reply:
Ah. OK. I didn't know `replace` takes a regex. But, shouldn't the `$` be a `^`?